model/Core/Classes/SpdxDocument.md

```diff
@@ -24,3 +24,8 @@ Commonly used when representing a unit of transfer of SPDX Elements.
   - minCount: 1
   - maxCount: 1
 
+## External properties restrictions
+
+- /Core/Element/name
```

Comment:
you use `/` as seperator and in other places we use `:`:

https://github.com/spdx/spdx-3-model/blob/01360996d37c1d913f3d15e2d5911ace182ed200/model/Software/Classes/File.md?plain=1#L16

Can we make this consistent?

Reply:
It would probably be best, to align with the way rdf is referencing the properties.